docs/js-core-2/cowsay-project/solutions/solution1.js

```diff
@@ -0,0 +1,42 @@
+// =================
+// Stripped down cowsayer CLI, 
+// no libraries
+// https://nodejs.dev/learn/nodejs-accept-arguments-from-the-command-line
+// =================
+
+// 1. Accept arguments
+
+const args = process.argv.slice(2);
+
+// 2. Make supplies for our speech bubble
+
+let topLine = '_';
+let bottomLine = '-';
+let saying = '';
+
+// 3. Make a cow that takes a string
+
+function cowsay(saying) {
+if(saying) {
+  let cowsaying =
+        `
+       ${topLine.repeat(saying.length)}
+     < ${saying} >
+       ${bottomLine.repeat(saying.length)}
```

Comment:
If we stripped these lines out and turned them into an exercise, I think that would make it a lot more approachable for JS1/2 students